teams/tests.py

```diff
@@ -0,0 +1,21 @@
+from .models import Teams
+import pytest
+
+
+@pytest.fixture
+def create_team():
+    team = Teams.objects.create(name='Test Team')
+    return team
+
+
+@pytest.mark.django_db
+class TestTeamsModel:
+
+    def test_create_team(self, create_team):
+        create_team.save()
+        assert create_team in Teams.objects.all()
```

Comment:
here it will be more readable like:
```suggestion
    def test_create_team(self, team1):
        team1.save()
        assert team1 in Teams.objects.all()
```